src/scripts/main.js

```diff
@@ -354,7 +354,21 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const myTable = document.querySelector('.dashboard tbody');
 
-// write your code here
+for (const person of people) {
+  const sexPerson = person.sex === 'm' ? 'Male' : 'Female';
+  const centuri = Math.ceil(person.died / 100);
```

Comment:
There is a typo in the variable name 'centuri'. It should be 'century' for better readability and clarity.